src/duke/ui/Ui.java

```diff
@@ -0,0 +1,209 @@
+package duke.ui;
+
+import java.util.Scanner;
+
+import duke.storage.TaskList;
+import duke.task.Task;
+
+/**
+ * Displays the user interface and deals with interacting with the user by printing responding to 
+ * user commands.
+ */
+public class Ui {
+    private static final String CHATBOT = "Bob: ";
+    private static final String SKIPLINE = "\n";
+    private static final String USER = SKIPLINE + "You: ";
+    private static final String DIVIDER = SKIPLINE + CHATBOT;
+    
+    public static boolean exit = false;
```

Comment:
Boolean Variables should be named to sound like booleans, consider renaming to isExited or some other such boolean sounding name